modules/core/src/v2/coins/eos.ts

```diff
@@ -22,6 +23,12 @@ interface AddressDetails {
   memoId: string;
 }
 
+interface EosTx {
```

Comment:
Since you're returning an object with a property of type `EosTx`, this should be exported since it's part of the public API.